pkg/util/file_extension.go

```diff
@@ -0,0 +1,9 @@
+package util
+
+func GetFileExtensionByContentType(contentType string) string {
+	extensions := map[string]string{
+		"text/html":  ".html",
+		"text/plain": ".txt",
+	}
+	return extensions[contentType]
```

Comment:
You should deal with the error case of an unrecognized content type.
```suggestion
	switch contentType {
		"text/html":
			return ".html"
		"text/plain": 
			return ".txt"
		default:
			panic(fmt.Sprintf("Unrecognized content type %q", contentType))
	}
```